src/themes/basic/_layout.styl

```diff
@@ -323,6 +323,8 @@ body.sticky
 
 .markdown-section iframe
   border 1px solid #eee
+  width 1px
+  min-width 100%
```

Comment:
Hi Trevor,
thank you for your contribution.

Do you mind to add a comment to explain that this is a fix for iOS safari.